docs/architecture/modern-web-apps-azure/index.md

```diff
@@ -82,6 +82,10 @@ More and more organizations are choosing to host their web applications in the c
 
 Building web applications with ASP.NET Core, hosted in Azure, offers many competitive advantages over traditional alternatives. ASP.NET Core is optimized for modern web application development practices and cloud hosting scenarios. In this guide, you'll learn how to architect your ASP.NET Core applications to best take advantage of these capabilities.
 
+## Version
+
+This guide has been revised to cover **.NET Core 3.1** version along with many additional updates related to the same “wave” of technologies (that is, Azure and additional third-party technologies) coinciding in time with the .NET Core 3.1 release. That’s why the book version has also been updated to version **3.1**.
```

Comment:
```suggestion
This guide has been revised to cover **.NET Core 3.1** version along with many additional updates related to the same "wave" of technologies (that is, Azure and additional third-party technologies) coinciding in time with the .NET Core 3.1 release. That's why the book version has also been updated to version **3.1**.
```

Reply:
There is tooling support for this: https://docs.microsoft.com/en-us/contribute/docs-authoring/smart-quote-replacement